src/main/java/sirius/biz/storage/layer3/FileQuickActionProvider.java

```diff
@@ -15,7 +15,7 @@
 import java.util.function.Consumer;
 
 /**
- * Represents a provider for file based quick actions.
+ * Represents a provider which provides link based quick actions which are associated with a file.
```

Comment:
I meant to use something like `{@linkplain VirtualFile file}`, because the word `file` is too generic.

Reply:
oh okay